kolibri/plugins/user_profile/tasks.py

```diff
@@ -82,5 +95,13 @@ def mergeuser(command, **kwargs):
 
     # Resync with the server to update the merged records
     call_command("sync", **kwargs)
+    new_superuser_id = kwargs.get("new_superuser_id")
+    if new_superuser_id:
+        new_superuser = FacilityUser.objects.get(id=new_superuser_id)
+        # make the user a new super user for this device:
+        new_superuser.facility.add_role(new_superuser, role_kinds.ADMIN)
```

Comment:
I wonder if this is necessary? Just the device permissions might be enough for the local device, and we avoid having another mechanism for adding an admin role to a facility (which could be a security risk). I don't think this is a blocker, but just wondering if we can avoid it.

Reply:
I am not sure this is needed either, but was a core part of the Figma design. Happy to remove the screen to select the new super user and this code if it's not really needed, but better to confirm it. I think it will be needed for the user to be able to add new content if the device is not an "Learner use only" device, aka _SOUD_